src/extension.ts

```diff
@@ -1109,9 +1114,15 @@ class ExtensionManager implements vscode.Disposable {
 
   selectLaunchTarget(folder?: vscode.WorkspaceFolder, name?: string) { return this.mapCMakeToolsFolder(cmt => cmt.selectLaunchTarget(name), folder); }
 
-  resetState(folder?: vscode.WorkspaceFolder) {
+  async resetState(folder?: vscode.WorkspaceFolder) {
     telemetry.logEvent("resetExtension");
-    return this.mapCMakeToolsFolder(cmt => cmt.resetState(), folder);
+    if (folder) {
+      await this.mapCMakeToolsFolder(cmt => cmt.resetState(), folder);
+    } else {
+      await this.mapCMakeToolsAll(cmt => cmt.resetState());
+    }
+
+    vscode.commands.executeCommand('workbench.action.reloadWindow');
   }
 
```

Comment:
Only the active folder was resetting state. I imagine that with this command, the user wants all projects to reset state. Without this change it was super annoying to test this PR in multi root.